specificatie/openapi.yaml

```diff
@@ -395,43 +395,43 @@ paths:
       tags:
         - Pand
       summary: "levert een pand"
-      description: "Raadpleeg een actueel pand met de identificatie. Een pand is een bouwkundige, constructief zelfstandige eenheid die direct en duurzaam met de aarde is verbonden en betreedbaar en afsluitbaar is. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.1.0/features/fields.feature)."
+      description: "Raadpleeg een actueel pand met de identificatie. Een pand is een bouwkundige, constructief zelfstandige eenheid die direct en duurzaam met de aarde is verbonden en betreedbaar en afsluitbaar is. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature)."
       parameters:
         - $ref: '#/components/parameters/pandidentificatie'
```

Comment:
In het endpoint '/adressen' is er voor gekozen om parameters zoals 'pandIdentificatie' in lowerCamelCase te definiëren waarom is daar hier bij 'pandidentificatie' niet voor gekozen?